setup.py

```diff
@@ -6,8 +6,8 @@
 
 
 def get_requirements():
-    """Parse all packages mentioned in the 'requirements.txt' file."""
-    with open('requirements.txt') as fd:
```

Comment:
Not sure if we really need this..? requirement.in outputs requirements.txt 
It's `.txt` which ideally be used for further processing. 
Why `.txt` coz it has a list of dependencies and transitive in flat list format.

Reply:
I did this cause it was recommended. @dgpatelgit any idea why?

Reply:
@deepak1725 We need this change as suggested by the initial check list provided by Arun for this migration task.

Reply:
@deepak1725, requirement.txt will have pinned versions which would cause conflicts. However this is not required in this repo because it is not a installable library. It is an application.